planner/solver.go

```diff
@@ -259,11 +260,13 @@ func FindBestPlanningSolutions(placeClusters [][]matching.Place, topSolutionsCou
 	priorityQueue := &graph.MinPriorityQueueVertex{}
 	deduplicatedPlans := make(map[string]bool)
 
+	for _, cluster := range placeClusters {
```

Comment:
is cluster equal to Place or []Place here? if it is the latter, the sorting complexity is not small right? because we are sorting all combinations of possible plans, which can be exponential?

Reply:
The cluster equals `[]Place`. The complexity is `O(NMlog(M))`, so it is not terrible. We are sorting individual lists not combinations.

Reply:
i see, the sorting is done on a list of resorts or restaurants. I assume N is the number of timeslots, and M is number of places available in that slot.